packages/react-native-bridge/android/react-native-bridge/build.gradle

```diff
@@ -65,25 +70,30 @@ dependencies {
 
     implementation "com.facebook.react:react-native:$rnVersion"
     implementation "org.wordpress-mobile:react-native-svg:${extractPackageVersion(packageJson, 'react-native-svg', 'dependencies')}"
-    implementation "com.github.wordpress-mobile:react-native-video:${extractPackageVersion(packageJson, 'react-native-video', 'dependencies')}"
+    // implementation "com.github.wordpress-mobile:react-native-video:${extractPackageVersion(packageJson, 'react-native-video', 'dependencies')}"
+    implementation "com.github.wordpress-mobile:react-native-video:update-for-0.69-SNAPSHOT"
```

Comment:
This needs to be updated once we released the new version of React Native Video.